src/components/Form.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Repo from "./Repo";
+import Greeting from "./Greeting";
+import "./form.css";
+
+class Form extends React.Component {
+  state = { nameInput: "" };
+
+  updateInput = (e) => {
+    const username = e.target.value;
+    this.setState({ nameInput: username });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    this.fetchGithubData(this.state.nameInput);
+    this.setState({ nameInput: "" });
+  };
+
+  fetchGithubData = (username) => {
+    const url = `https://api.github.com/users/${username}/repos`;
+    fetch(url)
+      .then((r) => r.json())
+      .then((data) =>
+        data.sort((a, b) => {
+          return new Date(b.updated_at) - new Date(a.updated_at);
+        })
+      )
+      .then((data) => this.setState({ userData: data }))
+      .catch((err) => {
+        console.warn(`Sorry but...${err}`);
+      });
+  };
```

Comment:
Your page works as it should which means your fetchGitHubData function works too. However, I personally don't understand what exactly it's doing. But great job anyway!